airflow/bin/cli.py

```diff
@@ -257,6 +258,48 @@ def variables(args):
         print(msg)
 
 
+@cli.action_logging
+def pools(args):
+    """
+    set a batch of Airflow pools
+    :param args: filepath to airflow pools yaml file
+    :return:
+    """
+    with open(args.filepath, 'r') as stream:
+        try:
+            pools_config = yaml.load(stream)
+        except yaml.YAMLError as exc:
+            print(exc)
+
+    session = settings.Session()
+    existing_pools = (
+        session.query(Pool)
+        .filter(Pool.pool.in_(pools_config.keys()))
+        .all())
+    for current_pool in existing_pools:
+        input_pool_slots = pools_config.get(current_pool.pool).get('slot_counts', 3)
+        input_pool_desc = pools_config.get(current_pool.pool).get('description', '')
+        if int(input_pool_slots) != int(current_pool.slots) or input_pool_desc != current_pool.description:
+            print("Need to update pool: {}".format(current_pool.pool))
+            print("Original slots: {}\nNew slots: {}".format(current_pool.slots, input_pool_slots))
+            print("Original description: {}\nNew description: {}".format(current_pool.description, input_pool_desc))
```

Comment:
in the above two print statements, the slots or description could be the same, and we will print. This is minor, but I just wanted to make sure we were aware.